src/templates/newengland-organizing.html

```diff
@@ -0,0 +1,102 @@
+                <h3 id="deciding-on-events">Deciding on events</h3>
+                <p>You definitely have some say over what events you want, but this competition should benefit the region too. Holding at least some beginner events, like 3x3, 2x2, and Pyraminx allows beginners to compete which is good for the community. It’s fine if you want to have less common events like 7x7, but the competition should be fun for everyone. In terms of scheduling, you should create a tentative schedule based on what you think is realistic. It’s good to add buffer time in case some events get behind. The Delegate will help you adjust the schedule to be more reasonable if needed.</p>
+                <h3 id="the-schedule">The schedule</h3>
+                <p>The organizer is responsible for creating the schedule, and the Delegate will approve it and make suggestions to try to accurately reflect realistic times. Please fill out the scheduling section of the spreadsheet at the top of this document when it’s time to make a schedule. While this will vary greatly depending on the competition, you should almost always have a new competitor tutorial at or around the beginning of the day before any popular events have started. We also suggest having some buffer time at some point in the day in case we get behind schedule. One way to do this is to make lunch longer than it needs to be so that it can also serve as a buffer should the competition fall behind.</p>
+                <h3 id="the-website">The website</h3>
+                <p>The Delegate will create a WCA website for your competition and send you a link. Be sure you’re signed in to view the site, since it won’t be public yet. You are responsible for filling in all relevant information, and then the Delegate will approve the site and send it to the WCA Competition Announcement Committee when it’s ready to announce. One of the decisions you’ll have to make is when to open and registration. You will generally want to leave a week or two between when the competition is announced and when registration opens to give everyone a chance to learn about the competition and decide whether they want to sign up. You should also leave some time between the closing of registration and the competition itself to give yourself time to generate and print the scorecards.</p>
```

Comment:
Can we just recommend specific timeframes?  e.g. 1 week after the competition is announced, and 1 week before the competition